voctocore/lib/commands.py

```diff
@@ -60,11 +87,38 @@ def set_composite_mode(self, composite_mode):
 		self.pipeline.vmix.setCompositeMode(mode)
 		return True
 
-	def set_stream_blank(self, src_name_or_id):
-		src_id = self.decodeBlankerSourceName(src_name_or_id)
-		self.pipeline.streamblanker.setBlankSource(src_id)
-		return True
+	def get_composite_mode(self):
+		try:
+			mode = self.pipeline.vmix.getCompositeMode()
+			name = str(mode).split('.')[-1]
```

Comment:
why is that split there? if it is required, shouldn't it go into vmix.getCompositeMode?